<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MorphologyWindow</class>
 <widget class="QMainWindow" name="MorphologyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>64</y>
      <width>771</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelRaw">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelAfter">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>核元素形状</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbMORPH_RECT">
       <property name="text">
        <string>MORPH_RECT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbMORPH_CROSS">
       <property name="text">
        <string>MORPH_CROSS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbMORPH_ELLIPSE">
       <property name="text">
        <string>MORPH_ELLIPSE</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>36</y>
      <width>771</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSlider" name="kSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPersent">
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>滤波</string>
    </property>
    <addaction name="actiondilate"/>
    <addaction name="actionerode"/>
    <addaction name="actionopen"/>
    <addaction name="actionclose"/>
    <addaction name="actionGradient"/>
    <addaction name="actionTopHat"/>
    <addaction name="actionBlackHat"/>
    <addaction name="actionHitMiss"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actiondilate">
   <property name="text">
    <string>dilate</string>
   </property>
  </action>
  <action name="actionerode">
   <property name="text">
    <string>erode</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actionGradient">
   <property name="text">
    <string>Gradient</string>
   </property>
  </action>
  <action name="actionTopHat">
   <property name="text">
    <string>TopHat</string>
   </property>
  </action>
  <action name="actionBlackHat">
   <property name="text">
    <string>BlackHat</string>
   </property>
  </action>
  <action name="actionHitMiss">
   <property name="text">
    <string>HitMiss</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
